int8 EStop
int8 index
int8 xMin
int8 xMax
int8 yMin
int8 yMax
int8 zMin
int8 zMax